src/main/java/com/google/api/generator/engine/ast/Statement.java

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.engine.ast;
+
+public interface Statement extends AstNode {
+  void accept(AstNodeVisitor visitor);
+}
```

Comment:
For AstNode interface, it also has one accept method, can we reuse it? Thanks

Reply:
Statement is a subtype of AstNode, and re-specifying the accept method here will let implementing classes clearly see all the methods that need to be implemented.